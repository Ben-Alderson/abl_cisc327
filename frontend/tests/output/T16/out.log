Enter command: Enter login: Enter command: Enter the account to withdraw: Enter amount to withdraw: You have successfully withdrawn $10.00 from 7488316
Enter command: Enter command: 