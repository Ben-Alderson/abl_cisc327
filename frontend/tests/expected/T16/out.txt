Enter command: Enter login: Enter command: Enter the account to withdraw: Enter amount to withdraw: You have successfully withdrawn $1000.00 from 7488316
Enter command: Enter command: 